cmd/uhc/account/status/cmd.go

```diff
@@ -91,4 +92,6 @@ func run(cmd *cobra.Command, argv []string) {
 	currAccount := response.Body()
 	fmt.Println(fmt.Sprintf("%s on %s", currAccount.Username(), cfg.URL))
 
```

Comment:
Please add a comment or use the one in line 91 to encompass what you are doing here.

Reply:
Gotcha, additional comment added.